docs_src/pipelinerun.md

```diff
@@ -207,21 +214,15 @@ The **results/pipeline0.log** file lists FPS during pipeline run.
 
 The **results** directory is volume mounted to the pipeline container. The log files within the **results** increase as the pipeline continues to run. You can [stop the pipeline](./pipelinerun.md#stop-pipeline-run) and the containers that are running.
 
-**Failure**
-    
-Review the console output for errors if you do not see all the Docker* containers. Sometimes dependencies fail to resolve. Address obvious issues and [rerun the pipeline](./pipelinerun.md#start-pipeline).
+!!! Failure
+    Review the console output for errors if you do not see all the Docker* containers. Sometimes dependencies fail to resolve. Address obvious issues and [rerun the pipeline](./pipelinerun.md#start-pipeline).
     
 ---
     
-### Stop Pipeline
+## Stop Pipelines
```

Comment:
should this remain as pipeline, as everywhere on the top we are referring to single pipeline ? or if we change it here then change ones on top to pipelines - eg `Start Pipelines` ?

Reply:
let me change it back.